tests-fuzz/tests/main.rs

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+
+use common_telemetry::info;
+use rand::{Rng, SeedableRng};
+use sqlx::mysql::MySqlPoolOptions;
+use sqlx::postgres::PgPoolOptions;
+use tests_fuzz::fake::{
+    merge_two_word_map_fn, random_capitalize_map, uppercase_and_keyword_backtick_map,
+    uppercase_and_keyword_quote_map, MapWordGenerator,
+};
+use tests_fuzz::generator::create_expr::CreateTableExprGeneratorBuilder;
+use tests_fuzz::generator::Generator;
+use tests_fuzz::translator::greptime::create_expr::CreateTableExprTranslator;
+use tests_fuzz::translator::DslTranslator;
+
+const GT_STANDALONE_MYSQL_ADDR: &str = "GT_STANDALONE_MYSQL_ADDR";
+const GT_STANDALONE_POSTGRES_ADDR: &str = "GT_STANDALONE_POSTGRES_ADDR";
+
+#[tokio::test]
```

Comment:
Not sure if it should be run as a unit test. Let take a look afterward

Reply:
Let me remove it